appengine/cloudsql/server.js

```diff
@@ -15,69 +15,87 @@
 
 'use strict';
 
+// Require process, so we can mock environment variables
+const process = require('process');
+
 // [START app]
 // [START setup]
 const express = require('express');
-const mysql = require('mysql');
+const Knex = require('knex');
 const crypto = require('crypto');
 
 const app = express();
 app.enable('trust proxy');
 // [END setup]
+let knex;
+
+function connectMysql () {
```

Comment:
I still think you only need one connection method. Just set `client` in the Knex options to `process.env.SQL_CLIENT`, then the docs just need to tell the user to set the `SQL_CLIENT` env var.